lib/modules/build-system/index.js

```diff
@@ -199,6 +200,19 @@ function BuildSystem(stageMethods, $q) {
             buildPath = path.join(basePath, '.fire', '.build', 'public', 'styles');
         }
 
+        var lessConfig = app.get('less');
```

Comment:
This should become `app.settings` actually.

`app.get` also exists, but is used for express-like middleware.